src/main/java/kitchenpos/ordertable/domain/TableGroup.java

```diff
@@ -0,0 +1,90 @@
+package kitchenpos.ordertable.domain;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class TableGroup {
+
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private LocalDateTime createdDate;
+
+    @Embedded
+    private OrderTables orderTables = new OrderTables();
+
+    public TableGroup() {
+    }
+
+    public TableGroup(LocalDateTime createdDate) {
+        this.createdDate = createdDate;
+    }
+
+    public TableGroup(Long id, LocalDateTime createdDate) {
+        this.id = id;
+        this.createdDate = createdDate;
+    }
+
+    public void groupTables(List<OrderTable> inputOrderTables) {
+        orderTables.groupTables(inputOrderTables);
+        for (OrderTable orderTable : inputOrderTables) {
+            orderTable.groupIn(this);
+            orderTable.changeOrderClose(false);
+        }
```

Comment:
orderTable에 그룹을 설정해주는 책임을 orderTables로 이전해보면 어떨까요 ?
```suggestion
        orderTables.groupTables(inputOrderTables);
```